chore: update readme.md

danielwernerio opened the pull request:
**- What I did**

Improvement of the representation of NanoCl in context of the README.md.

**- How I did it**

My own personal thoughts.

**- How to verify it**

Please read it and give me feedback.

**- Description for the changelog**

Optimized readme.md.

danielwernerio committed:
chore: update readme.md

Signed-off-by: Daniel Werner <82284479+danielwernerio@users.noreply.github.com>

danielwernerio committed:
chore: update readme.md

Improved:

- Naming (Nanocl, Statefiles)
- Style
- Minor text stuff

Signed-off-by: Daniel Werner <82284479+danielwernerio@users.noreply.github.com>

0xle0ne commented on README.md:
```diff
@@ -1,79 +1,73 @@
 ## Installation
 
-To install **Nanocl** for your system, please refer to our online [installation guide][nanocl_install_guide].
+We are already compatible with **Debian/Ubuntu, MacOS & Windows**, just jump to [Installation guide][nanocl_install_guide].<br/>
```
Can you replace Debian/Ubuntu with linux ?

danielwernerio replied:
This has to be corrected within the documentation, too.

0xle0ne replied:
Yeap will do during the day

danielwernerio replied:
Perfect. Leo is a real hustler.

0xle0ne reviewed (changes requested):
A last thing before merging

danielwernerio committed:
chore: update readme.md

Optimization:
- Installation section: Replaced "Debian/Ubuntu" with "Linux" (more general). 

--> The documentation has to be updated afterwards.

Signed-off-by: Daniel Werner <82284479+danielwernerio@users.noreply.github.com>
